examples/mosquitto/snapcraft.yaml

```diff
@@ -0,0 +1,37 @@
+name: mosquitto
+version: 0.1
+summary: mosquitto server and client
+description: |
+  MQTT example with a server, a publisher and a subscriber.
+apps:
+  mosquitto:
+    command: usr/sbin/mosquitto -c $SNAP/mosquitto.conf
+    daemon: simple
+    caps:
+      - network-listener
+      - network-service
+    security-override:
+      read-paths:
+        - /etc/hosts.deny
+        - /etc/hosts.allow
+  subscribe:
+    command: bin/subscribe
+    caps:
+      - network-listener
+  publish:
+    command: bin/publish
+    caps:
+      - network-listener
+
+parts:
+    mosquitto:
+      plugin: copy
```

Comment:
You have some inconsistent spacing throughout here-- sometimes three spaces, sometimes two.